src/component/race/Race.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import octane from "./images/octane.png";
+import "./css/Race.css";
+
+function Race(props) {
+  const wordsToAdvance = Math.round(props.wordsLength / 15);
```

Comment:
What significance does 15 have?

Reply:
@jgayfer you can think of that number as 'positions' in the race. It could be saved into a variable; it's a limiter on how fast or slow you want the icon to move.